### ISSUE TITLE
Feature Request: Teams Option in Apps

### ISSUE THREAD

**Issue by dscott3d** [veloce/lichobile#1165]
# Feature request: Teams Option in Apps
- Use case: players who only use the app cannot access the "Community > Teams" menu to see their teams, and enter Teams Only Tournaments, etc.
- Currently the workaround is to join the "Teams Only Tournament" via a browser on a desktop, then once in a game, switch to the app. Be great if this was added so it could all be done on the phone!

**Comment by SVVredeburg** [comment]
I totally agree. I organised a team tournament for which people needed to be a member of the team. But I received 5 calls/WhatsApp messages from people who couldn't find them in the app. Sure, they can use the site, but it would save a bit of bother.

**Comment by BrendanEMahon** [comment]
Second @SVVredeburg. People expect the app to have full functionality now and don’t want to log in to the website on their phone. Would be a nice addition.

**Comment by MikePageDev** [comment]
I would like to see this implemented as I play a lot on my mobile.
